### ISSUE TITLE
[4.0] Wrong Message of com_messages

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#29169]
### Steps to reproduce the issue
In a local environment (where I don't have mail) wirte a private message.
The message is stored in the database and the receiver gets the "new message" badge on top right.

### Expected result
Info: Messages is sent
Warning: The mail funtion does not work

### Actual result
![screen shot 2020-05-20 at 12 30 03](https://issues.joomla.org/uploads/1/99ad4d40997f1ceb4d698c0bd18c89b4.jpg)

**Comment by brianteeman** [comment]
Technically the messages are correct as they refer to the sending of the email notification of the message - i think

**Comment by chmst** [comment]
Yes. But "Save failed " here I thought that it is not stored at all.

**Comment by brianteeman** [comment]
True - that is misleading

**Comment by brianteeman** [comment]
The only way to "fix" this is to overide the save function in `libraries\src\MVC\Controller\FormController.php` in the com_messages controller - I think
